scripts/get_git_commit.py

```diff
@@ -0,0 +1,19 @@
+import subprocess
+import os
+
+revision = "00000000000000000000000000000000000000000"
+
+env_rev = os.environ.get("GIT_REV")
+if env_rev != "":
+    revision = env_rev
+
+try:
+    revision = (
+        subprocess.check_output(["git", "rev-parse", "HEAD"])
+        .strip()
+        .decode("utf-8")
+    )
+except Exception:
+    revision = "NOT_GIT"
```

Comment:
```suggestion
revision = ""

env_rev = os.environ.get("GIT_REV")
if not env_rev is None and env_rev != "":
    revision = env_rev
else:
	try:
		revision = (
			subprocess.check_output(["git", "rev-parse", "HEAD"])
			.strip()
			.decode("utf-8")
		)
	except Exception:
		revision = "NOT_GIT"
```

Reply:
Mixed tabs and spaces

Reply:
oops I've just typed within the browser ∑0w0. It's fixed now